cmd/utils/flags.go

```diff
@@ -976,6 +976,18 @@ var (
 		Category: flags.RollupCategory,
 		Value:    true,
 	}
+	RollupSequencerEnableTxConditionalFlag = &cli.BoolFlag{
+		Name:     "rollup.sequencerenabletxconditional",
+		Usage:    "Serve the eth_sendRawTransactionConditional endpoint and apply the conditional constraints on mempool inclusion & block building",
+		Category: flags.RollupCategory,
+		Value:    false,
+	}
+	RollupSequencerTxConditionalRateLimitFlag = &cli.Uint64Flag{
```

Comment:
```suggestion
	RollupSequencerTxConditionalRateLimitFlag = &cli.IntFlag{
```
So that it matches the type of the config